### DISCUSSION TITLE
ESP32 FLASH Tool

### DISCUSSION THREAD

**Discussion by 3aau** [tbnobody/OpenDTU#1094] [Q&A]
moin, irgendwie habe ich es geschaft mir einen esp32 in den e-haven zu flashen.
brauche mal nachhilfe bei der bedienung...

1.) bootloader.bin,  1000
2.) boot_app0.bin   8000
3.) partions.bin       E000
4.) opendtu-generic.bin 10000

40Mhz
DOUT
CombineBin
Default

stimmt das so?

da kommt dann:
NO XMC flash detecdet!
FLASH CRYPT_CNT 0
ABS_DONE_0 false
--------
KeyError: 255

**Comment by tbnobody** [comment]
Nein, nimm die .factory Datei und flashe sie auf Adresse 0x0.
Die Versionen mit den 4 einzelnen Dateien gibt es nicht mehr.

**Comment by 3aau** [comment]
ändert leider nix an der fehlermeldung...
ich habe die opendtu-generic_esp32.factory genommen.

**Comment by 3aau** [comment]
ok, nach dem gefühlt 20 versuch ging es dann doch.
hat wohl ne macke das ding.
danke für die hilfe.
